components/table/index.css

```diff
@@ -110,7 +110,7 @@ governing permissions and limitations under the License.
   /* Passthrough for nested component(s) */
   --mod-thumbnail-size: var(--mod-table-thumbnail-size, var(--spectrum-table-thumbnail-size));
 
-	[dir="rtl"] &,
+	&:dir(rtl),
 	&:dir(rtl) {
```

Comment:
Duplicate here
```suggestion
	&:dir(rtl) {
```

Reply:
Great call-out - fixed this duplication in both instances and pushed up